src/index.js

```diff
@@ -0,0 +1,67 @@
+import './main.css';
+import App from './modules/app.js';
+
+const form = document.querySelector('form');
+const refresh = document.querySelector('#refresh');
+
+document.addEventListener('DOMContentLoaded', async () => {
+  App.displayData();
+});
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  App.postData();
+});
+
+refresh.addEventListener('click', (e) => {
+  e.preventDefault();
+  App.displayData();
+});
+
+// const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BT0cx2WqtrqYvdaTYVfQ/scores/';
+
+// async function fetchScores() {
+//   const response = await fetch(baseURL);
+//   return response.json();
+// }
+
+// async function postScore(score) {
+//   const response = await fetch(baseURL, {
+//     method: 'POST',
+//     headers: {
+//       'Content-Type': 'application/json',
```

Comment:
Please ensure that you remove all the commented codes, they make your code dirty.